Automattic/wp-calypso
# Add/help center launchpad navigation v1

#### Proposed Changes

* Added HelpCenter entry point to Launchpad screen if `launchpad_screen` option is `full`

#### Testing Instructions



* Checkout this branch
* Sandbox public API and a launchpad enabled site
* Navigate to Calypso `apps/editing-toolkit` and execute `yarn dev --sync`
* Navigate to the site-editor and click the question mark icon on the top right navigation bar to open the help center.
* Verify you are able to see the Launchpad entry point and that it navigates to the launchpad screen correctly.
* Test this out on another screen such as the `/posts` page.

<img width="439" alt="image" src="https://user-images.githubusercontent.com/10482592/215506244-f2c76a2f-3b7a-432e-bebd-11947a10fa6b.png">

Hover effect:

<img width="382" alt="image" src="https://user-images.githubusercontent.com/10482592/215519692-4e61a5b9-5543-4d43-8ab3-c455b53174df.png">


#### Pre-merge Checklist





Related to #72276
